### PR TITLE
Add initial adopters file

### PR DESCRIPTION
**Pull request by afrittoli**
Add an adopters file where open source projects, vendores and users can tell the community about their involvement with Tekton.

Partially fixes: #804

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

File: adopters.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Tell us about, we welcome [pull requests](https://github.com/tektoncd/community/edit/main/adopters.md)!
+
+## Open Source Projects
+
+| Project Name | Description | Repository |
+|--------------|-------------|---------|
+| [Shipwright](https://shipwright.io/)   | Shipwright is an extensible framework for building container images on Kubernetes. | [https://github.com/shipwright-io](github.com/shipwright-io)
+| [Jenkins X](https://jenkins-x.io/)    | All In One CI/CD including everything you need to start exploring Kubernetes | [github.com/jenkins-x](https://github.com/jenkins-x) |
+| kfp-tekton | Kubeflow Pipelines SDK for Tekton | [github.com/kubeflow/kfp-tekton](https://github.com/kubeflow/kfp-tekton/) |
+| OpenShift Pipelines | OpenShift Pipelines is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. | [github.com/openshift-pipelines](https://github.com/openshift-pipelines) |
```
Comment (adambkaplan) [inline adopters.md:16]: cc @vdemeester.

The `openshift-pipelines` repo is a catch-all for the things we are building on top of Tekton.

Comment (vdemeester) [reply adopters.md:16]: Yes the `openshift-pipelines` does indeed contains more than just OpenShift Pipelines, but as @adambkaplan said, it contains "on top of" Tekton things.

Comment (afrittoli) [reply adopters.md:16]: If you would like to provide more granularity here that works for me, we can either merge it and you patch it later or let me know what link/components you would like to mention in the table :)

**Comment by babadofar** [comment]
👍

**Pull request merged**